### ISSUE TITLE
MariaDB delete statements are generated with aliases that are not supported

### ISSUE THREAD

**Issue by 04nd01** [Budibase/budibase#13189]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**

- Self
  - Method: docker single image
  - Budibase Version: 2.21.0

**Describe the bug**
when attempting to delete a row from a MariaDB (version 11.3.2) table, Budibase will generate and execute the following query:
```
delete from `sets` as `a` where `a`.`id` = 11
```
Within an app in design view, this leads to an error message referring to the location of the query the alias is defined in. In the data source management view it gets stuck on a loading animation after confirming.

According to my tests, aliases can't be used in MariaDB delete statements and thus the delete fails. running the generated query directly on the MariaDB database also fails, manually running the same query with the aliases removed is successful
```
delete from `sets`  where `sets`.`id` = 11
```

**To Reproduce**
Add any MariaDB Table to an app and try deleting a row from the table within Budibase.

**Expected behavior**
Queries with statements that do not support aliases should be generated without them.

**Comment by 04nd01** [comment]
The issue is also present on version 2.21.1 and recent commits indicate that changes to SQL aliasing have been made, maybe one of those changes is what caused this issue.

**Comment by mike12345567** [comment]
This should be resolved as of 2.21.2
